tools/gdb/Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG AGENT_VERSION
+FROM datadog/agent:${AGENT_VERSION}
+
+# Install gdb
+# Use the debian snapshot repo commented out in the sources.list file, so that we install
+# the same versions of the debug packages (that are deps of gdb) as what's originally installed in the image
+RUN grep '^\#' /etc/apt/sources.list | sed 's/# //' > /etc/apt/sources.list.tmp \
+ && mv /etc/apt/sources.list.tmp /etc/apt/sources.list \
+ && apt-get -o Acquire::Check-Valid-Until=false update \
+ && apt-get install -y gdb
+
+# Install Go version used to build the Agent, to get appropriate GDB debug script
+RUN export GIMME_GO_VERSION=$(agent version | grep --only-matching -E 'go[0-9\.]*' | sed 's/go//') \
+ && eval "$(curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | bash)"
+
+# Install datadog-agent-dbg at the right version
+ARG AGENT_VERSION
+RUN apt-get install -o Dpkg::Options::="--force-confold" --no-install-recommends -y gnupg dirmngr ca-certificates \
+ && echo 'deb https://apt.datadoghq.com/ stable 7 6' > /etc/apt/sources.list.d/datadog.list \
```

Comment:
We should also add support for non-stable channels and non production repos.

Reply:
Fixed, let me know what you think @truthbk